### PR TITLE
OPENEUROPA-1322: Use the proper Docker way of overriding docker-composer.yml file.

### PR DESCRIPTION
**Pull request by drupol**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,76 +1,134 @@
...
 
 * Customize build settings by copying `runner.yml.dist` to `runner.yml` and
 changing relevant values, like your database credentials.
 
 * Setup test site by running:
 
-```
-$ ./vendor/bin/run drupal:site-setup
+```bash
+./vendor/bin/run drupal:site-setup
```
Comment (dxvargas) [inline README.md]: This should be removed like in other repositories we are now updating.

File: README.md [hunk 2]
```diff
@@ -1,76 +1,134 @@
...
-$ docker-compose exec web ./vendor/bin/behat
-```
+
+## Contributing
+
+Please read [the full documentation](https://github.com/openeuropa/openeuropa) for details on our code of conduct, and the process for submitting pull requests to us.
+
+## Versioning
+
+We use [SemVer](http://semver.org/) for versioning. For the available versions, see the [tags on this repository](https://github.com/openeuropa/epoetry/tags).
```
Comment (dxvargas) [inline README.md]: Wrong link, it must refer to oe_paragraphs.

File: README.md (hunk 1 above)
Comment (drupol) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (drupol) [reply README.md]: fixed.

Comment (richardcanoe) [reply README.md]: Now it's fixed!

**Pull request merged**